src/api/parser/src/data/post.js

```diff
@@ -151,7 +151,7 @@ class Post {
     }
 
     // Wrap an <a> tag on any link inside our content
-    article.content = linkifyHtml(article.content);
+    article.content = linkifyHtml(article.content, { ignoreTags: ['img'] });
```

Comment:
Good job, I think you need to remove the current temporary fix for this as well. cc @416rehman

Reply:
You can safely remove this if this works
https://github.com/Seneca-CDOT/telescope/blob/c29eec4be6cee6aaf6ade6186fa7c0942d61bc2d/src/web/src/components/Posts/Post.tsx#L330

Reply:
@DukeManh just upload another commit

Reply:
nice catch